### PR TITLE
Payments: add missing include type

### PR DESCRIPTION
**Pull request by janpaepke**

### CODE REVIEW

**Review by Pimm — APPROVED** [review]
Turning `PaymentInclude` into an enum is consistent with most of our existing API, but a discussion is to be had [about using enums vs string constants](https://github.com/mollie/mollie-api-node/issues/284).

**Comment by Pimm** [comment]
Fixes #264.

**Comment by janpaepke** [comment]
> Fixes #264.

How does this fix #264 ? That one is about `PaymentLink`, this one is about `PaymentInclude`.

File: src/data/payments/data.ts [hunk 1]
```diff
@@ -862,7 +862,7 @@ export enum PaymentStatus {
 
 export enum PaymentInclude {
   qrCode = 'details.qrCode',
-  remainderDEtails = 'details.remainderDetails',
```
Comment (janpaepke) [inline src/data/payments/data.ts]: woops 😅

**Pull request merged**